cognite/neat/core/utils.py

```diff
@@ -156,11 +158,48 @@ def retry_decorator(max_retries=2, retry_delay=3, component_name=""):
     def decorator(func):
         @wraps(func)
         def wrapper(*args, **kwargs):
+            previous_exception = None
             for attempt in range(max_retries + 1):
                 try:
                     logging.debug(f"Attempt {attempt + 1} of {max_retries + 1} for {component_name}")
                     return func(*args, **kwargs)
+                except CogniteReadTimeout as e:
+                    previous_exception = e
+                    if attempt < max_retries:
+                        logging.error(
+                            f"""CogniteReadTimeout retry attempt {attempt + 1} failed for {component_name} .
+                            Retrying in {retry_delay * 2} second(s). Error:"""
+                        )
+                        logging.error(e)
+                        time.sleep(retry_delay * 2)
```

Comment:
Why are you multiplying with 2 here? This is setting the retry delay twice as long as the signature of the function indicates